2025/09/14 18:13:13 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii_appointment_booking.categoryiesc' doesn't exist. The SQL statement executed was: SELECT category.name, COUNT(*) as count
FROM `appointment` `a`
JOIN `categoryiesc` ON a.category_id = c.id
WHERE a.user_id = :user_id
GROUP BY `c`.`name`. Bound with :user_id='5'.
in C:\xampp\htdocs\appointment-booking\protected\controllers\UserController.php (164)
in C:\xampp\htdocs\appointment-booking\index.php (13)
2025/09/14 18:13:13 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yii_appointment_booking.categoryiesc' doesn't exist. The SQL statement executed was: SELECT category.name, COUNT(*) as count
FROM `appointment` `a`
JOIN `categoryiesc` ON a.category_id = c.id
WHERE a.user_id = :user_id
GROUP BY `c`.`name`. Bound with :user_id='5' in C:\xampp\htdocs\yii-sample\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-sample\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\appointment-booking\protected\controllers\UserController.php(164): CDbCommand->queryAll()
#2 [internal function]: UserController->actionDetails('5')
#3 C:\xampp\htdocs\yii-sample\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(UserController), Array)
#4 C:\xampp\htdocs\yii-sample\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii-sample\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii-sample\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii-sample\framework\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii-sample\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii-sample\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii-sample\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii-sample\framework\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii-sample\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii-sample\framework\web\CWebApplication.php(282): CController->run('details')
#14 C:\xampp\htdocs\yii-sample\framework\web\CWebApplication.php(141): CWebApplication->runController('user/details')
#15 C:\xampp\htdocs\yii-sample\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\xampp\htdocs\appointment-booking\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/appointment-booking/user/details?userId=5
HTTP_REFERER=http://localhost/appointment-booking/user/list
---
2025/09/14 18:13:37 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'category.name' in 'field list'. The SQL statement executed was: SELECT category.name, COUNT(*) as count
FROM `appointment` `a`
JOIN `categories` ON a.category_id = c.id
WHERE a.user_id = :user_id
GROUP BY `c`.`name`. Bound with :user_id='5'.
in C:\xampp\htdocs\appointment-booking\protected\controllers\UserController.php (164)
in C:\xampp\htdocs\appointment-booking\index.php (13)
2025/09/14 18:13:37 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'category.name' in 'field list'. The SQL statement executed was: SELECT category.name, COUNT(*) as count
FROM `appointment` `a`
JOIN `categories` ON a.category_id = c.id
WHERE a.user_id = :user_id
GROUP BY `c`.`name`. Bound with :user_id='5' in C:\xampp\htdocs\yii-sample\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-sample\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\appointment-booking\protected\controllers\UserController.php(164): CDbCommand->queryAll()
#2 [internal function]: UserController->actionDetails('5')
#3 C:\xampp\htdocs\yii-sample\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(UserController), Array)
#4 C:\xampp\htdocs\yii-sample\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii-sample\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii-sample\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii-sample\framework\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii-sample\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii-sample\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii-sample\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii-sample\framework\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii-sample\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii-sample\framework\web\CWebApplication.php(282): CController->run('details')
#14 C:\xampp\htdocs\yii-sample\framework\web\CWebApplication.php(141): CWebApplication->runController('user/details')
#15 C:\xampp\htdocs\yii-sample\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\xampp\htdocs\appointment-booking\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/appointment-booking/user/details?userId=5
HTTP_REFERER=http://localhost/appointment-booking/user/list
---
2025/09/14 18:14:12 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'c.name' in 'field list'. The SQL statement executed was: SELECT c.name, COUNT(*) as count
FROM `appointment` `a`
JOIN `categories` `categories` ON a.category_id = c.id
WHERE a.user_id = :user_id
GROUP BY `c`.`name`. Bound with :user_id='5'.
in C:\xampp\htdocs\appointment-booking\protected\controllers\UserController.php (164)
in C:\xampp\htdocs\appointment-booking\index.php (13)
2025/09/14 18:14:12 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'c.name' in 'field list'. The SQL statement executed was: SELECT c.name, COUNT(*) as count
FROM `appointment` `a`
JOIN `categories` `categories` ON a.category_id = c.id
WHERE a.user_id = :user_id
GROUP BY `c`.`name`. Bound with :user_id='5' in C:\xampp\htdocs\yii-sample\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii-sample\framework\db\CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\appointment-booking\protected\controllers\UserController.php(164): CDbCommand->queryAll()
#2 [internal function]: UserController->actionDetails('5')
#3 C:\xampp\htdocs\yii-sample\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(UserController), Array)
#4 C:\xampp\htdocs\yii-sample\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UserController), Object(ReflectionMethod), Array)
#5 C:\xampp\htdocs\yii-sample\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii-sample\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii-sample\framework\web\filters\CFilter.php(40): CFilterChain->run()
#8 C:\xampp\htdocs\yii-sample\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii-sample\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#10 C:\xampp\htdocs\yii-sample\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#11 C:\xampp\htdocs\yii-sample\framework\web\CController.php(291): CFilterChain->run()
#12 C:\xampp\htdocs\yii-sample\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\yii-sample\framework\web\CWebApplication.php(282): CController->run('details')
#14 C:\xampp\htdocs\yii-sample\framework\web\CWebApplication.php(141): CWebApplication->runController('user/details')
#15 C:\xampp\htdocs\yii-sample\framework\base\CApplication.php(185): CWebApplication->processRequest()
#16 C:\xampp\htdocs\appointment-booking\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/appointment-booking/user/details?userId=5
HTTP_REFERER=http://localhost/appointment-booking/user/list
---
2025/09/14 20:30:26 [error] [exception.CException] CException: Property "User.status" is not defined. in C:\xampp\htdocs\yii-sample\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii-sample\framework\db\ar\CActiveRecord.php(145): CComponent->__get('status')
#1 C:\xampp\htdocs\appointment-booking\protected\views\site\profile.php(47): CActiveRecord->__get('status')
#2 C:\xampp\htdocs\yii-sample\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\yii-sample\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii-sample\framework\web\CController.php(872): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii-sample\framework\web\CController.php(785): CController->renderPartial('profile', Array, true)
#6 C:\xampp\htdocs\appointment-booking\protected\controllers\SiteController.php(145): CController->render('profile', Array)
#7 C:\xampp\htdocs\yii-sample\framework\web\actions\CInlineAction.php(49): SiteController->actionProfile()
#8 C:\xampp\htdocs\yii-sample\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii-sample\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii-sample\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii-sample\framework\web\CWebApplication.php(282): CController->run('profile')
#12 C:\xampp\htdocs\yii-sample\framework\web\CWebApplication.php(141): CWebApplication->runController('site/profile')
#13 C:\xampp\htdocs\yii-sample\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\xampp\htdocs\appointment-booking\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/appointment-booking/site/profile
HTTP_REFERER=http://localhost/appointment-booking/site/index
---
2025/09/14 20:31:24 [error] [exception.CException] CException: Property "User.status" is not defined. in C:\xampp\htdocs\yii-sample\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii-sample\framework\db\ar\CActiveRecord.php(145): CComponent->__get('status')
#1 C:\xampp\htdocs\appointment-booking\protected\views\site\profile.php(47): CActiveRecord->__get('status')
#2 C:\xampp\htdocs\yii-sample\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\yii-sample\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii-sample\framework\web\CController.php(872): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii-sample\framework\web\CController.php(785): CController->renderPartial('profile', Array, true)
#6 C:\xampp\htdocs\appointment-booking\protected\controllers\SiteController.php(145): CController->render('profile', Array)
#7 C:\xampp\htdocs\yii-sample\framework\web\actions\CInlineAction.php(49): SiteController->actionProfile()
#8 C:\xampp\htdocs\yii-sample\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii-sample\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii-sample\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii-sample\framework\web\CWebApplication.php(282): CController->run('profile')
#12 C:\xampp\htdocs\yii-sample\framework\web\CWebApplication.php(141): CWebApplication->runController('site/profile')
#13 C:\xampp\htdocs\yii-sample\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\xampp\htdocs\appointment-booking\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/appointment-booking/site/profile
HTTP_REFERER=http://localhost/appointment-booking/site/index
---
2025/09/14 20:31:46 [error] [exception.CHttpException.404] CHttpException: The system is unable to find the requested action "profilePicture". in C:\xampp\htdocs\yii-sample\framework\web\CController.php:486
Stack trace:
#0 C:\xampp\htdocs\yii-sample\framework\web\CController.php(270): CController->missingAction('profilePicture')
#1 C:\xampp\htdocs\yii-sample\framework\web\CWebApplication.php(282): CController->run('profilePicture')
#2 C:\xampp\htdocs\yii-sample\framework\web\CWebApplication.php(141): CWebApplication->runController('site/profilePic...')
#3 C:\xampp\htdocs\yii-sample\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 C:\xampp\htdocs\appointment-booking\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/appointment-booking/site/profilePicture/5
HTTP_REFERER=http://localhost/appointment-booking/site/profile
---
